tutorials/evals/evaluate_rag_haystack.ipynb

```diff
@@ -0,0 +1,1690 @@
+{
```

Comment:
<p>You don't need the OTEL_EXPORTER_OTLP_HEADERS if you use register. Also this doesn't seem to have support for running your own phoenix? Feels like our tutorials should make affordances for that rather than tutorials being one way or another</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>That or we move this under hosted_phoenix for now</p>